File: Algorithms/Iteration/Bubble Sort/javascripts/bubblesort.js
function bubbleSort(array, size){
  for(let i=0; i<size-1; i++){
    for(let j= i+1;j< size;j++){
      if (array[i]>array[j]) {
        // swap array[i] & array[j]
        let temp = array[i];
        array[i] = array[j];
        array[j] = temp;
      }
    }
  }
  }

  let data =[23,11,6,15,1,3];
  let size = data.length;
  bubbleSort(data,size);
  console.log(data);

File: Algorithms/Iteration/Bubble Sort/java/Bubblesort.java
/*Program for bubble sort in java*/
package javacode.sort;

import java.util.Arrays;

public class Bubblesort {

	public void bubble_sort(int array[], int size) {
		int i, j;
		for (i = 0; i < size - 1; i++) {
			for (j = i + 1; j < size; j++) {
				if (array[i] > array[j]) {
					// swap array[i] & array[j]
					int temp = array[i];
					array[i] = array[j];
					array[j] = temp;

				}
			}

		}

	}

	public static void main(String[] args) {
		int[] data = { 3, 11, 23, 6, 1, 15 };
		Bubblesort bubblesort = new Bubblesort();
		int size = data.length;
		bubblesort.bubble_sort(data, size);
		System.out.println(Arrays.toString(data));
	}

}
